src/services/pointService.js

```diff
@@ -0,0 +1,38 @@
+import { errorMessage, successMessage } from "./alert.service";
+import { generalErrorMessages } from "../constants/messages/general";
+import { pointErrorMessages } from "../constants/messages/point";
+import pointService from "../api/point";
+
+export function createPoint(values, history) {
+    var model = {
```

Comment:
You can simply send values without creating a model, if they have the same fields